### ISSUE TITLE
add time / meta info to SMC / meta info samplers in general

### ISSUE THREAD

**Issue by florianhartig** [florianhartig/BayesianTools#152]
SMC doesn't record the time - adjust the SMC to record the same info as all the samplers.

In general, I think it would be good to have an extra slot in the samplers to record meta info, such as runtime, operating system, packages etc (MCMCs record most of this already, but in settings)

**Comment by MaximilianPi** [comment]
I think, this was the intention of our "info" field, right?

**Comment by florianhartig** [comment]
yes, right, I had the feeling that there was something going on there - so this is  https://github.com/florianhartig/BayesianTools/issues/118

**Comment by MaximilianPi** [comment]
I think we had some issues about what we should move to the info field and it was postponed

**Comment by florianhartig** [comment]
I'm trying to finalize 0.1.5 now and we should do this for 0.1.6
